### PR TITLE
Added value 'DCP/2 Ingest' to enum for file source. Fixes #1487.

### LINKED ISSUE #1487: Add "HCA Ingest" as a value for "file_core.file_source" field
**For which schema is a change/update being suggested?**

I would like to update `file_core.json`

**What should the change/update be?**

This ticket is part of the effort to expose a metadata spreadsheet to the consumers of the data in the HCA Data portal, as described in the [PR 66](https://github.com/HumanCellAtlas/dcp2/pull/66) update.

### PR DESCRIPTION
**Pull request by ESapenaVentura**
### Release notes

For `file_core` schema:
- Added value to enum for file source
 

### Reviews requested

- This is a minor change to all the schemas it affects (file_core + file type schemas)

### CODE REVIEW

File: json_schema/core/file/file_core.json [hunk 1]
```diff
@@ -64,7 +64,8 @@
                 "DCP/1 Matrix Service",
                 "LungMAP",
                 "Zenodo",
-                "Publication"
+                "Publication",
+                "HCA Ingest"
```
Comment (hannes-ucsc) [inline json_schema/core/file/file_core.json:68]: Having second thought on this. Should this be

```suggestion
                "DCP/2 Ingest"
```

Comment (hannes-ucsc) [reply json_schema/core/file/file_core.json:68]: … considering the existing `DCP/1 Matrix Service`, that is.

Comment (ESapenaVentura) [reply json_schema/core/file/file_core.json:68]: That sounds good to me, I will commit the change!

File: json_schema/core/file/file_core.json [hunk 2]
```diff
@@ -64,7 +64,8 @@
                 "DCP/1 Matrix Service",
                 "LungMAP",
                 "Zenodo",
-                "Publication"
+                "Publication",
+                "DCP/2 Ingest"
             ],
             "user_friendly": "File source",
             "guidelines": "Should be one of: DCP/2 Analysis, Contributor, ArrayExpress, HCA Release, GEO, SCEA, SCP, DCP/1 Matrix Service, LungMAP, Zenodo, Publication",
```
Comment (ncalvanese1) [inline json_schema/core/file/file_core.json:71]: Should the guidelines be updated to reflect the updated enum?

Comment (ESapenaVentura) [reply json_schema/core/file/file_core.json:71]: You mean https://github.com/HumanCellAtlas/dcp2/pull/66?

Comment (ncalvanese1) [reply json_schema/core/file/file_core.json:71]: I actually meant the "guidelines" property in the json here. It seems like it lists the enums (which in itself may be a bit redundant), but it wasn't updated the include the additional enum value.

Comment (hannes-ucsc) [reply json_schema/core/file/file_core.json:71]: I'll update the spec PR.

File: docs/jsonBrowser/core.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ file_name | The name of the file. | string | yes |  | File name |  | R1.fastq.gz
 format | The format of the file. | string | yes |  | File format |  | fastq.gz; tif
 content_description | General description of the contents of the file. | array | no | [See module  file_content_ontology](module.md#file-content-ontology) | Content description |  | 
 checksum | MD5 checksum of the file. | string | no |  | Checksum |  | e09a986c2e630130b1849d4bf9a94c06
-file_source | The source of the file. This is typically an organisation, repository, person or dedicated process. | string | no |  | File source | DCP/2 Analysis, Contributor, ArrayExpress, HCA Release, GEO, SCEA, SCP, DCP/1 Matrix Service, LungMAP, Zenodo, Publication | GEO; Contributor
+file_source | The source of the file. This is typically an organisation, repository, person or dedicated process. | string | no |  | File source | DCP/2 Analysis, Contributor, ArrayExpress, HCA Release, GEO, SCEA, SCP, DCP/1 Matrix Service, LungMAP, Zenodo, Publication, HCA Ingest | GEO; Contributor
```
Comment (hannes-ucsc) [inline docs/jsonBrowser/core.md:21]:
```suggestion
file_source | The source of the file. This is typically an organisation, repository, person or dedicated process. | string | no |  | File source | DCP/2 Analysis, Contributor, ArrayExpress, HCA Release, GEO, SCEA, SCP, DCP/1 Matrix Service, LungMAP, Zenodo, Publication, DCP/2 Ingest | GEO; Contributor
```

**Review by hannes-ucsc — CHANGES_REQUESTED** [review]

**Review by amnonkhen — APPROVED** [review]

File: json_schema/core/file/file_core.json (hunk 2 above)
Comment (ESapenaVentura) [reply json_schema/core/file/file_core.json:71]: Oh, apologies @ncalvanese1 I read this as a standalone comment and did not realise you were referring to the "guidelines" field. 

I will address this comment and Hannes' suggestion on the markdown file

**Review by ncalvanese1 — APPROVED** [review]
Looks good to me, thanks!

**Review by hannes-ucsc — APPROVED** [review]

**Review by NoopDog — APPROVED** [review]

File: json_schema/update_log.csv [hunk 4]
```diff
@@ -1 +1,2 @@
 Schema,Change type,Change message,Version,Date
+core/file/file_core,minor,"Added value to enum for file source. Fixes #1487.",,
```
Comment (amnonkhen) [inline json_schema/update_log.csv:2]: Do we need a value in the date column?

**Review by amnonkhen — APPROVED** [review]

File: json_schema/update_log.csv (hunk 4 above)
Comment (ESapenaVentura) [reply json_schema/update_log.csv:2]: No, we don't! it gets filled automatically by the `src/release_prepare.py` script. We just need an empty value

**Pull request merged**